packages/ses/src/lockdown-shim.js

```diff
@@ -164,7 +163,13 @@ export function repairIntrinsics(options = {}) {
   return hardenIntrinsics;
 }
 
-export const lockdown = (options = {}) => {
-  const maybeHardenIntrinsics = repairIntrinsics(options);
-  return maybeHardenIntrinsics();
+export const makeLockdown = makeCompartmentConstructor => {
```

Comment:
`makeLockdown` is designed so it can be called with no parameters. But that is not clear from the code here. I suggest that, whenever possible, we use optional parameter syntax for parameters intended to be option, even if that has no semantic effect. Here
```js
(makeCompartmentConstructor = undefined) => {
```